apps/web/server/routers/viewer.tsx

```diff
@@ -574,8 +574,8 @@ const loggedInViewerRouter = createProtectedRouter()
       // `flatMap()` these work like `.filter()` but infers the types correctly
       const conferencing = apps.flatMap((item) => (item.variant === "conferencing" ? [item] : []));
       const payment = apps.flatMap((item) => (item.variant === "payment" ? [item] : []));
+      const other = apps.flatMap((item) => (item.variant.startsWith("other") ? [item] : []));
```

Comment:
The `startsWith` is needed as the variant for apps closely related to calendars is `other_calendar` so it can be treated as a calendar when creating bookings but not shown in the "Calendars" section in the Installed Apps page, which involve having a selected calendar and the `listCalendars` that does not apply to HubSpot in this case.